### ISSUE TITLE
Shred with Bleachbit doesn't work when the application is installed only for the current user

### ISSUE THREAD

**Issue by rados** [bleachbit/bleachbit#1166]
1. Install for the current user (Windows). 
2. Try 'Shred with Bleachbit' context menu command over a file
3. Nothing happens

Expected behavior: the file that we execute the command on should be shredded.

**Comment by sanjaykaria** [comment]
> 2\. Try 'Shred with Bleachbit' context menu command over a file

@rados 
I cannot find the context menu in Windows explorer. Have installed on Windows 10 machine the latest stable build [https://www.bleachbit.org/download/file/t?file=BleachBit-4.4.2-setup.exe](url)

Can you please clarify this issue?

**Comment by rados** [comment]
@sanjaykaria I have it:

![image](https://user-images.githubusercontent.com/332322/218981082-3a2f598a-bd85-484e-84a7-b9b6979730c9.png)

**Comment by sanjaykaria** [comment]
@rados 
Can you please share a bit larger screenshot?

**Comment by sanjaykaria** [comment]
@rados

The "Shred with BleachBit" context menu in Windows Explorer is installed by the BleachBit installer by creating a registry entry at "HKEY_CLASSES_ROOT\AllFilesystemObjects\shell\shred.bleachbit\command".

However, admin rights are needed to create an entry at the above location. When BB is installed for the current user (assuming the current user is a Standard user, without admin rights ) the Shell integration will not occur. Consequently, the "Shred with BleachBit" context menu in Windows Explorer will not be available.

Can you please confirm whether the use case you describe above occurs, after ensuring that the Shell integration is explicitly removed and then BB is installed for current user (without admin rights)?

**Comment by sanjaykaria** [comment]
@az0 

I tried installing and using the app with different combinations of Standard and Admin user (as below) but, I do not see any issue in any of the use cases.

![image](https://user-images.githubusercontent.com/3830310/219572606-d6a72e29-175a-4d16-8653-a9f63520322c.png)

Not sure if am missing something.

**Comment by rados** [comment]
@sanjaykaria, you are right, I confirm - no context menu command when the app is installed only for the current user.

**Comment by az0** [comment]
Next step: investigate use of registry `HKCU\Software\Classes`, which does not require admin rights.
